server/routes/events.js

```diff
@@ -55,7 +55,7 @@ router.patch('/:id', checkJwt, (req, res) => {
     })
 })
 
-// doesnt need token
+// TODO fix req.user.admin
```

Comment:
This is a tricky one because it should be able to be used by all 3 types of users: not authenticated, authenticated garden members, and garden admins. And how the route behaves depends on the user. I haven't yet figured out how to get the user information if a token is provided but still allow the route to be accessed if a token isn't provided. Not yet sure what to do here 🤔